src/styles/blocks/card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 198px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+
+}
+
+.card__description {
+  margin-bottom: 4px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__id {
+  margin-bottom: 16px;
+
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-description {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-amount {
+
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy-link {
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #fff;
+  text-decoration: none;
+
+  background: #00acdc;
+  border-radius: 5px;
+
+}
+
+.card__buy-link:hover {
+  color: #00acdc;
+
+  background: #fff;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+
+  transition: background-color 0.5s, color 0.5s;
```

Comment:
Don't define transition in the pseudoclass, because these rules could apply for another state (focus, active, click, etc)